### ISSUE TITLE
[Question] Is mm_projector parameter freeze?

### ISSUE THREAD

**Issue by unmo** [haotian-liu/LLaVA#832]
### Question

[finetune_task.sh](https://github.com/haotian-liu/LLaVA/blob/main/scripts/v1_5/finetune_task.sh) does not have the --mm_projector_lr parameter, does this mean that mm_projector is frozen?
Thank you.

**Comment by Linziyang1999** [comment]
--mm_projector_lr is a parament to change learning rate of m_projector, its not mean whether tune mm_projector or not, its not frozen by default.

**Comment by unmo** [comment]
Thank you.
According to the following implementation, the default is None, but how many lr would be in that case?
https://github.com/haotian-liu/LLaVA/blob/main/llava/train/train.py#L105C5-L105C20

**Comment by Linziyang1999** [comment]
You can check llama/train/llava_trainer.py there is a customized optimizer
